game-core/src/main/java/games/strategy/triplea/ai/proAI/simulate/ProDummyDelegateBridge.java

```diff
@@ -97,7 +96,8 @@ public ISound getSoundChannelBroadcaster() {
   }
 
   @Override
-  public void enterDelegateExecution() {}
+  public void enterDelegateExecution() {
+  }
```

Comment:
Seems like your formatter isn't configured correctly.

Reply:
It's there, these were hand updates that were going down another branch that I backed out. I applied reformat to this file in: d588c63